CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Deprecations
 
+* `ui.diff.format` and `ui.diff.tool` were deprecated in favor of `ui.diff-formatter`
```

Comment:
Not totally convinced that `ui.diff-formatter` is the best setting name. Maybe `ui.diff-viewer` would be better?

Reply:
"viewer" sounds like a GUI or TUI, but it's actually a command to generate textual diff.

Reply:
Can you not configure it to open a GUI or TUI though? (Not something I've done but I've seen discussion about it). I don't have a strong preference, so will go with `ui.diff-viewer` as @martinvonz is OK with it.

Reply:
Yes, GUI works. TUI wouldn't because tty isn't accessible.

I don't have a strong opinion either. If "viewer" works, I don't mind.

Reply:
When I read `diff-viewer`, I assumed this was a setting to configure the diff tool that opens when you do `jj diff --tool NAME`. If it's for formatting a diff that will be written to stdout, then `diff-formatter` seems more appropriate to me.